archetypes/business-service-pojo/src/main/resources/archetype-resources/README.md

```diff
@@ -0,0 +1,20 @@
+## Welcome to use ServiceComb Java Chassis
+This project(module) is generate by *org.apache.servicecomb.archetypes:business-service-pojo-archetype*, it use **pojo provider** to develop service producer.  
+
+### More works can be done further:
+1. Modify "HelloImpl", add your business service logic, or create some new producers to provide your services. More details can be found : http://servicecomb.incubator.apache.org/users/develop-with-transparent-rpc/
+2. Modify "microservice.yaml", change APPLICATION_ID, service_description.name, version, and service center address, endpoints publish address etc. More details can be found : http://servicecomb.incubator.apache.org/users/service-definition/
+3. Modify setting value of "mainClass" in pom.xml for package.
+
+### Package your service
+Under project(module) root folder, run 
+```bash
+mvn package
+```
+Then you can get outputs in target folder:   
+- lib : contains all dependencies jars   
+- xxxxxx-{version}.jar   
+```bash
+java -jar xxxxxx-{version}.jar
+```
+*Notice: If you need to modify config setting in "microservice.yaml" like service center address but don't want repackage the executable jar, **you can direct place a new "microservice.yaml" file in same folder, then settings will be overridden.***
```

Comment:
config setting -> configurations
direct ->directly
in same folder ->under the same folder
settings -> configurations

Reply:
Done,Thanks